### DISCUSSION TITLE
DISCUSSION_TEMPLATE not work for non-English category

### DISCUSSION THREAD

**Discussion by shenweiyan** [community/community#87430] [Discussions]
### Select Topic Area

Bug

### Body

For example, I have a Chinese category named `测试`. I created `.github/DISCUSSION_TEMPLATE/测试.yml` in the repository, but when I try to create a discussion for the `测试` category, the predefined `测试.yml` template does not take effect.

**Comment by faridnec** [comment]
Perhaps GitHub Discussions might have difficulty recognizing and applying the template when the category name or template file includes non-ASCII characters, such as Chinese characters in your case (测试).

you can try by renaming your template file to (for instance) `.github/DISCUSSION_TEMPLATE/testing.yml` or any other name using English characters. GitHub tends to handle ASCII characters more reliably, and this might help with the recognition of the template! Or maybe if there is, inside the testing.yml file, update any references to the category name. Make sure it is referenced in a way that GitHub Discussions can recognize, even if it's a non-ASCII character.

**Reply by shenweiyan** [reply]
Thank you. It indeed might be an issue with non-ASCII characters, but I haven't seen any official documentation about it. I renamed my category name with English characters, and everything seems to be back to normal.

**Reply by faridnec** [reply]
These docs might help [Creating discussion category forms](https://docs.github.com/en/discussions/managing-discussions-for-your-community/creating-discussion-category-forms) & [Syntax for discussion category forms](https://docs.github.com/en/discussions/managing-discussions-for-your-community/syntax-for-discussion-category-forms).

If it is solved then nice!

**Comment by shenweiyan** [comment]
> These docs might help [Creating discussion category forms](https://docs.github.com/en/discussions/managing-discussions-for-your-community/creating-discussion-category-forms) & [Syntax for discussion category forms](https://docs.github.com/en/discussions/managing-discussions-for-your-community/syntax-for-discussion-category-forms).这些文档可能有助于创建讨论类别表单和讨论类别表单的语法。
> 
> If it is solved then nice!如果解决了那就太好了！

For Chinese characters, I haven't found any solution at the moment. I still hope that GitHub's DISCUSSION_TEMPLATE can provide support for non-English categories.
